android-kit-base/src/test/kotlin/com/mparticle/kits/KitManagerImplTest.kt

```diff
@@ -602,6 +602,127 @@ class KitManagerImplTest {
         Assert.assertTrue(manager.providers.containsKey(idOne))
     }
 
+    @Test
+    fun shouldFilterKitsFromKnownIntegrations() {
+        val options = MParticleOptions.builder(MockContext()).build()
+        val filteredKitOptions = MParticleOptions.builder(MockContext())
+            .disableKits(listOf(MParticle.ServiceProviders.ADJUST))
+            .disableKits(listOf(MParticle.ServiceProviders.APPBOY))
+            .disableKits(listOf(MParticle.ServiceProviders.CLEVERTAP))
+            .build()
```

Comment:
Is this API consistent with the Android SDK options?  I was expecting instead of 3 lines of 
```
.disableKits(listOf(MParticle.ServiceProviders.ADJUST))
.disableKits(listOf(MParticle.ServiceProviders.APPBOY))
.disableKits(listOf(MParticle.ServiceProviders.CLEVERTAP))
```

to instead be more like iOS where you would just drop a single list like:

something like
```
.setFilteredKits(Arrays.asList(
        MParticle.ServiceProviders.ADJUST,
        MParticle.ServiceProviders.APPBOY,
        MParticle.ServiceProviders.CLEVERTAP
    ))
    .build();
```